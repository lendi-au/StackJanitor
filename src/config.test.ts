describe("config", () => {
  test("it shoud match with the default config", () => {
    const config = require("./config");
    expect(config).toMatchSnapshot();
  });
  test.skip("dry run mode from environment", () => {
    process.env.DRY_RUN = "true";
    const config2 = require("../src/config");
    expect(config2.default.DRY_RUN).toEqual("true");
  });
});
