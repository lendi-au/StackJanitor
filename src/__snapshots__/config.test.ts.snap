// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config it shoud match with the default config 1`] = `
Object {
  "default": Object {
    "DEFAULT_DYNAMODB_TABLE": "stackJanitorTable",
    "DEFAULT_EXPIRATION_PERIOD": 604800,
    "DEFAULT_REGION": "ap-southeast-2",
    "DELETE_INTERVAL": 300,
    "DRY_RUN": false,
    "MAX_CLEANUP_RETRY": 5,
  },
}
`;
